doc/source/data/examples/index.rst

```diff
@@ -44,6 +44,12 @@ Batch inference
 
             Object Detection Batch Inference with PyTorch FasterRCNN_ResNet50
 
+    .. grid-item-card::
+
+        .. button-ref:: xgboost-example-ref
+
+            Tabular Data Batch Inference with XGBoost
```

Comment:
Thoughts on just merging this with the other one and renaming it to indicate that it's for training and inference?